hail/python/hailtop/utils/utils.py

```diff
@@ -136,7 +136,7 @@ def generator():
     return generator()
 
 
-def unzip(lst: List[Any]) -> Tuple[Any]:
+def unzip(lst: Iterable[Tuple[Any, Any]]) -> Tuple[List(Any),List(Any)]:
```

Comment:
Almost perfect except this `Tuple[List(Any),List(Any)]` should be this `Tuple[List[Any], List[Any]]`